src/main/java/org/apache/commons/io/TaggedIOException.java

```diff
@@ -90,6 +93,7 @@ public static boolean isTaggedWith(final Throwable throwable, final Object tag)
      * @param tag tag object
      * @throws IOException original exception from the tagged decorator, if any
      */
+    @SuppressWarnings("nullness:throwing.nullable") 
```

Comment:
Java doc for `TaggedIOException.getCause()` states that method has narrower return type. Return type for `Throwable.getCause()` can be `Nullable`. So I added `@Nullable` annotated in overridden getCause method which led to `throwing.nullable` warning. What does "narrower return type" imply here ?